### PR TITLE
Update Gitea to v1.16.5

### PR DESCRIPTION
**Pull request by Wuszek**
## Description

Keep gitea up to date -  v1.16.5

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created
- [x] Package upgrade completed successfully

### Type of change

- [x] Package update

### CODE REVIEW

File: spk/gitea/Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-SPK_REV = 3
+SPK_VERS = 1.16.4
+SPK_REV = 4
 SPK_ICON = src/$(SPK_NAME).png
 
 MAINTAINER = publicarray
 DESCRIPTION  = Gitea is a community managed lightweight code hosting solution written in Go.
 DISPLAY_NAME = Gitea
-CHANGELOG = "1. Making setting up SSH easier.<br/> 2. Update to v1.16.1.<br/> 3. Update to v1.16.3."
+CHANGELOG = "1. Making setting up SSH easier.<br/> 2. Update to v1.16.1.<br/> 3. Update to v1.16.3.<br/> 4. Update to v1.16.4."
```
Comment (hgy59) [inline spk/gitea/Makefile:9]: please avoid cumulative change log.
The change log should describe the changes to the latest published package only.

Comment (Wuszek) [reply spk/gitea/Makefile]: Fixed

**Comment by Wuszek** [comment]
@publicarray would you be so kind and merge it? :)

**Comment by hgy59** [comment]
> @publicarray would you be so kind and merge it? :)

propose to wait until go 1.18 is merged  #5152 (currently waiting for syncthing is fixed for go 1.18)

**Pull request merged**